src/common/platform_info.ts

```diff
@@ -24,14 +25,15 @@ const platformInfo = {
   isSnap: p.env.ELECTRON_SNAP,
   isDevelopment: process.env.NODE_ENV !== 'production',
   isAppImage: p.env.DESKTOPINTEGRATION === 'AppImageLauncher',
+  isDevEnv: isDevEnv,
   sshAuthSock: p.env.SSH_AUTH_SOCK,
   environment: process.env.NODE_ENV,
   debugEnabled: !!process.env.DEBUG,
   platform: easyPlatform,
   darkMode: testMode? true : e.nativeTheme.shouldUseDarkColors || windowPrefersDarkMode,
   userDirectory,
   testMode,
-  appDbPath: path.join(userDirectory, 'app.db'),
+  appDbPath: path.join(userDirectory, isDevEnv ? 'app.db' : 'app-dev.db'),
```

Comment:
isn't this backwards? isDevEnv === isPackaged, but shouldn't it be reversed?

Reply:
your right, fixed!